frontend/src/components/Main/index.tsx

```
@@ -351,16 +352,22 @@ const KFTray = () => {
       } catch (error) {
-        errors.push({ id: config.id, error })
+        return { id: config.id, error }
       }
-    }
+    })
+
+    const results = await Promise.allSettled(portForwardingPromises)
+
+    const errors = results
+    .map(result => (result.status === 'fulfilled' ? result.value : null))
+    .filter(result => result && result.error) as { id: number; error: any }[]
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)